tests/integration/test_textarea.py

```diff
@@ -27,3 +31,15 @@ def test_acceptable_submission(self):
 
         self.post(action=None)
         self.assertInUrl('thank-you')
+
+    def test_big_list_of_naughty_strings(self):
+        self.launchSurvey('test', 'big_list_naughty_strings')
+
+        answers = {}
+        for i in range(0, len(NAUGHTY_STRINGS)):
+            k = 'answer{}'.format(i)
+            answers[k] = NAUGHTY_STRINGS[i]
```

Comment:
What happens if new strings get added? it won't be able to find the answer index e.g. 508?